lib/compress/zstd_compress.c

```diff
@@ -2455,6 +2455,7 @@ static size_t ZSTD_compressBlock_targetCBlockSize(ZSTD_CCtx* zc,
                                const void* src, size_t srcSize,
                                U32 lastBlock) {
     size_t cSize = 0;
+    int compressSuperBlock = 1;
```

Comment:
I believe the name of this variable is slightly misleading.
`compressSuperBlock` seems to underline a link with `SuperBlock`, but if I do understand the code properly, there is no such direct link, as this variable only tells if the generated block is compressed or not.
Thus it should rather be called `compressedBlock` or something equivalent.

Reply:
Ah yes. good point